elastic/integrations
# [IIS] Update document with supported ingest patterns for `access_log`

## What does this PR do?

The IIS access_log provides flexibility for the customer to edit the default logging fields and add new fields. The IIS integration currently supports the default and a few custom selections. The supported patterns are captured in the document.

## How to test this PR locally

Verify the documentation in the access_log section and make sure that the supported patterns are captured.

## Related issues

- Closes #6709
